packages/arb-token-bridge-ui/package.json

```diff
@@ -43,13 +43,15 @@
     "swr": "^2.1.2",
     "tailwind-merge": "^0.9.0",
     "use-query-params": "^2.1.1",
+    "viem": "^1.16.6",
     "wagmi": "^0.12.18",
     "zod": "^3.22.2",
     "zustand": "^4.3.9"
   },
   "scripts": {
+    "predev": "yarn wagmi:generate",
```

Comment:
NIT: Should we add this in project running instructions as well?

Reply:
it's run automatically when run `yarn dev` so i don't think it needs to be added